Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -42,10 +42,19 @@ protected class MemoryCacheItem
             public BitmapImage Image { get; set; }
         }
 
+        private class ConcurrentTask
+        {
+            public Task<BitmapImage> Task { get; set; }
+
+            public bool IsAssureOnly { get; set; }
```

Comment:
I wouldn't find IsPreCache on the file but i think PreCacheAsync or EnsureCachedAsync are both okay.

I do however strongly think that once we merge these changes, someone should go throught the file and clean it up..  maybe streamline it a bit